interpreter/cling/lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -420,7 +421,10 @@ namespace {
     R.clear();
     R.setLookupName(&m_Context->Idents.get("copyArray"));
     m_Sema->LookupQualifiedName(R, NSD);
-    assert(!R.empty() && "Cannot find cling::runtime::internal::copyArray");
+    /* If I am not being called from a child Interpreter, only then do
```

Comment:
Would this be ok?

// FIXME: In the case of the multiple interpreters (parent-child), 
 // the child interpreter doesn't include the runtime universe. 
 // The child interpreter will try to import this function from its 
 // parent interpreter, but it will fail, because this is a template function.
  // Once the import of template functions becomes supported by clang,
  // this check can go away.